Packs/CommonScripts/Scripts/FormatURL/FormatURL_test.py

```diff
@@ -325,6 +325,29 @@ def test_get_redirect_url_from_query_no_url_query_param(self, mocker):
         assert get_redirect_url_from_query(url_, urlparse(url_), 'q') == url_
         assert demisto.error.called
 
+    @pytest.mark.parametrize('url_, expected', [
+        ('[https://urldefense.com/v3/__https://google.com:443/search?66ujQIQ$]',
+         'https://urldefense.com/v3/__https://google.com:443/search?66ujQIQ$'),
+        ('(https://urldefense.us/v3/__https://google.com:443/searchERujngZv9UWf66ujQIQ$)',
+         'https://urldefense.us/v3/__https://google.com:443/searchERujngZv9UWf66ujQIQ$'),
+        ('[someURL].', 'someURL'),
+        ('[another_test)', '[another_test)'),
```

Comment:
So in this case no bracket will be removed?

Reply:
No, unless we do not distinguish between the type of the special character. Maybe let's discuss what we expect to do.